app/controllers/api/memberships_controller.rb

```diff
@@ -24,7 +36,7 @@ def autocomplete
     @memberships = Queries::VisibleAutocompletes.new(query: params[:q],
                                                      group: @group,
                                                      current_user: current_user,
-                                                     limit: 10)
+                                                     limit: 5)
```

Comment:
I tend to want to include more rather than less because then there is less lag as the user gets more accurate